src/Helper/CommandExecutor.php

```diff
@@ -140,21 +140,41 @@ private function readAlternating(array $descriptors)
             stream_set_blocking($descriptor, false);
             $outputs[$key] = '';
         }
+        $nretries = 6;
```

Comment:
Can you rename `$nretries` and `$nresources` to `$retries` and `$resources` please?